Docs: nested tables (#9436)
ckeditor/ckeditor5: packages/ckeditor5-table/docs/features/table.md

	By default, table styling tools are not included in the {@link builds/guides/overview ready–to–use editor builds} and must be installed separately. See the [installation](#table-and-cell-styling-tools-2) section to learn how to enable them in your editor.
</info-box>

### Nesting tables

> Starting from version 27.1.0 CKEditor 5 allows nesting tables inside other table's cells. This may be used for creating advanced charts or layouts based on tables. The nested table can formatted just like a regular one.

Reviewer: ```suggestion
Starting from version 27.1.0 CKEditor 5 allows nesting tables inside other table's cells. This may be used for creating advanced charts or layouts based on tables. The nested table can be formatted just like a regular one.
```

> Test this feature by adding a table in the *"abandoned"* section that was left blank at the bottom of the main table. To nest a table, simply click in the selected cell and use the **"Insert table"** button in the toolbar to insert a new, nested table into an existing one.

Reviewer: ```suggestion
You can test this feature in the demo below by adding a table in the *"abandoned"* section that was left blank at the bottom of the main table. To nest a table, simply click in the selected cell and use the **"Insert table"** button in the toolbar to insert a new, nested table into an existing one.
```

{@snippet features/table-nesting}

<info-box>

> 	If you would want to disallow nesting tables in your editor, see the {@link features/table#disallow-nesting-tables Disallow nesting tables} section to learn how to disable this functionality.

Reviewer: ```suggestion
    If you would like to block the possibility to nest tables in your editor, see the {@link features/table#disallow-nesting-tables Disallow nesting tables} section to learn how to disable this functionality.
```

	.then( ... )
	.catch( ... );
```
<info-box>

> 	Check the {@link framework/guides/creating-simple-plugin plugin development} guide if you need more information about the technical side of this solution.

Reviewer: ```suggestion
	Check the {@link framework/guides/creating-simple-plugin plugin development guide} if you need more information about the technical side of this solution.
```

Author: We used to leave the and guide outside the link?

	We recommend using the official {@link framework/guides/development-tools#ckeditor-5-inspector CKEditor 5 inspector} for development and debugging. It will give you tons of useful information about the state of the editor such as internal data structures, selection, commands, and many more.
</info-box>

## Known issues

> While the tables nesting functionality is fully functional, it is not yet supported with some output features. Feel free to upvote 👍&nbsp; these issues on GitHub if they are important for you:

Reviewer: ```suggestion
While the table nesting functionality is fully functional, it is not yet supported with some output features. Feel free to upvote 👍&nbsp; these issues on GitHub if they are important for you:
```

> * [#9474](https://github.com/ckeditor/ckeditor5/issues/9474) &ndash; The `.docx` file generated with the [Export to Word](https://ckeditor.com/docs/ckeditor5/latest/features/export-word.html) feature will not open properly if it contains nested tables.

Reviewer: ```suggestion
* [#9474](https://github.com/ckeditor/ckeditor5/issues/9474) &ndash; The `.docx` file generated with the [export to Word](https://ckeditor.com/docs/ckeditor5/latest/features/export-word.html) feature will not open properly if it contains nested tables. Due to this, an alert will be shown if you attempt to export a file containing a nested table to Word.
```

> * [#9475](https://github.com/ckeditor/ckeditor5/issues/9475) &ndash; The Markdown code generated with the {@link features/autoformat Markdown output} feature will not properly render code for nested tables.

Reviewer: ```suggestion
* [#9475](https://github.com/ckeditor/ckeditor5/issues/9475) &ndash; The Markdown code generated with the {@link features/autoformat Markdown output} feature will not properly render nested tables.
```
